template/en/default/admin/params/admin.html.tmpl

```diff
@@ -23,6 +23,14 @@
    desc = "Set up account policies"
 %]
 
+[% rate_limit_rules_desc = BLOCK %]
+This parameter is a json object that allows for overriding the default rate limit.
+Each key takes an array in the form of [MAX_RATE, SECONDS] where MAX_RATE is the maximum
+number of requests that can occur over SECONDS. The default is [75, 60] or 75 requests
```

Comment:
The syntax description above does not offer any way for the user to select the default.